src/app/core/metric-module/metric.service.ts

```diff
@@ -47,10 +47,22 @@ export class MetricService {
    * @returns metrics for one learning object or all released learning objects
    */
   // TODO: This isn't being referenced anywhere and may not be needed
-  async getLearningObjectMetrics(cuid?: string) {
+  async getLearningObjectMetrics(cuid?: string, start?: string, end?: string) {
+    let route = METRIC_ROUTES.GET_LEARNING_OBJECT_METRICS(cuid);
+    const params: string[] = [];
+    if (start) {
+      params.push(`start=${encodeURIComponent(start)}`);
+}
+    if (end) {
+      params.push(`end=${encodeURIComponent(end)}`);
+}
+    if (params.length) {
+      route += (route.includes('?') ? '&' : '?') + params.join('&');
+    }
+
```

Comment:
It looks like this isn't doing anything different in our backend, our backend route for learning objects metrics has a DTO that only accepts the learning object cuid, adding start and end on the client doesn't change the results coming back.